fields/types/password/PasswordType.js

```diff
@@ -123,10 +123,17 @@ password.prototype.validateInput = function (data, callback) {
 	var result = true;
 	var confirmValue = this.getValueFromData(data, '_confirm');
 	var passwordValue = this.getValueFromData(data);
-	if (passwordValue !== undefined && confirmValue !== undefined && passwordValue !== confirmValue) {
+	if (passwordValue !== undefined
+		&& confirmValue !== undefined
+		&& passwordValue !== confirmValue) {
 		result = false;
 		detail = 'passwords must match';
 	}
+	if ((passwordValue === undefined || confirmValue === undefined)
```

Comment:
This isn't necessary - if the `confirmValue` isn't provided, we don't validate it server-side. Password fields may be updated programatically or after client-side verification,